bsdlib/doc/CHANGELOG.rst

```diff
@@ -5,6 +5,23 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.7.0
+************
+
+* Major rewrite of the lower transport layer to fix an issue where packages where lost in a high bandwidth application.
+* Added support for GPS priority setting to give the GPS module priority over LTE to generate a fix.
+* Added parameter checking and only return -1 on error for the PDN set socket option function.
+* Added support for send timeout on TCP, UDP (including secure sockets), and AT sockets.
+* Added support for MSG_TRUNC on AT, GNSS, TCP, and UDP sockets.
+* Allocating more sockets than available will now return ENOBUFS instead of ENOMEM.
+* GNSS socket now supports stopping a stopped GPS module to apply a delete mask.
```

Comment:
"stopping a stopped GPS module" -> is this how it's supposed to be?

Reply:
Yes, a bit strange sentence, but it has to do with the state machine in the GNSS driver. Before you where not allowed to try to transition to STOPPED state if you where already in STOPPED state. The reason you want to to do that is to apply a delete mask without having to transition to STARTED and then back to STOPPED mode if you only want to apply this delete mask.